### PR TITLE
PD-Node-W4

### PR DESCRIPTION
**Pull request by esma-g**
## What does this change?

Module: Node
Week(s): 4

## Checklist

- [x] I have read the [contributing guidelines](https://syllabus.codeyourfuture.io/contributing/overview)
- [x] I have checked my spelling and grammar with an [automated tool](https://www.grammarly.com/grammar-check)
- [x] I have previewed my changes to check the [markdown renders](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) as I intend
- [x] I have run my code to check it works
- [x] My changes follow our [Style Guide](https://syllabus.codeyourfuture.io/guides/code-style-guide)

## Description

- Created a page for Week 4 and added PD lesson
- Added week 4 links to the sidebars and index
- Deleted coursework sentence as it doesn't apply to this week

## Who needs to know about this?

@kfklein15 @CodeYourFuture/syllabus-team 

## Rendered Pages
https://personal-development.codeyourfuture.io/em-modules/node/week-4-leadership-compass

### CODE REVIEW

File: docs/node/week-4/homework.md
Comment (illicitonion) [inline docs/node/week-4/homework.md]: I don't currently see a week 4 folder, or a PR adding one - am I missing this, or is happening in the future? Do we want to block this PR on adding that?

File: docs/node/week-4/lesson.md
Comment (illicitonion) [inline docs/node/week-4/lesson.md:3]:
```suggestion
title: Node - 4
```

**Review by illicitonion — COMMENTED** [review]
LGTM, but looks like the homework content is missing?

**Comment by esma-g** [comment]
> LGTM, but looks like the homework content is missing?

Yes, the homework content has not been developed yet. I was only creating the page for it.

File: docs/node/week-4/homework.md
Comment (esma-g) [reply docs/node/week-4/homework.md]: > I don't currently see a week 4 folder, or a PR adding one - am I missing this, or is happening in the future? Do we want to block this PR on adding that?

This is similar to #620 and #638. Week 4 will need to be linked, could you help with that @illicitonion?

Comment (illicitonion) [reply docs/node/week-4/homework.md]: For getting the node week 4 page linked, hopefully you can crib from https://github.com/CodeYourFuture/syllabus/pull/638 - you need to edit sidebars.js to add to the left hand side, and docs/node/index.md to add a new table row. (If you're having trouble, I can help out, but hopefully you can crib that ok yourself :))

But the comment I was leaving here is a bit different - the other repo (https://github.com/CodeYourFuture/NodeJS-Homework) doesn't have a "week 4" folder - I'd expect us to add one there before merging this PR (or alternatively, to remove the homework.md in this PR, and only add it in a future PR when the homework exists). What do you think?

Comment (esma-g) [reply docs/node/week-4/homework.md]: Alright, I'll try to crib your example to link the pages. Let's see how I fare :)

Re homework repo: Sorry, now I understand what you meant. We have had the same situation for JS3, right? [The JS3 homework repo](https://github.com/CodeYourFuture/JavaScript-Core-3-Homework) doesn't have a week 4, but we have already merged [PD week 4 PR](https://github.com/CodeYourFuture/syllabus/pull/620).

I think this is happening because syllabus pages for JS3, React, Node, and SQL still have only 3 weeks. In practice, all modules take place over 4 weeks. Again in practice, SQL is called databases... 

If adding a folder is as straightforward as copying an existing example, I'm happy to add week 4 folder to those modules (I can try to crib the NodeJS repo).

Comment (esma-g) [reply docs/node/week-4/homework.md]: Update: maybe I was looking to the wrong place. There seems to be a repo for [JS3 Week 4](https://github.com/CodeYourFuture/JavaScript-Core-3-Coursework-Week4), as well as for [Node Week 4](https://github.com/CodeYourFuture/Node-Coursework-Week4).

But I'm not sure what this means...

**Comment by esma-g** [comment]
Hi @illicitonion! I'm back to this task. I think I sorted it out this time.

I linked the new pages to the sidebar/index and deleted the coursework sentence as it doesn't apply to this week. Could you review it one more time please? 😊

**Comment by esma-g** [comment]
@illicitonion, @kfklein15 Could I get a review for this one, please?

**Review by kfklein15 — APPROVED** [review]
ok

**Pull request merged**
